packages/esm-amd-loader/tsconfig.json

```diff
@@ -7,8 +7,9 @@
     "preserveConstEnums": false,
     "sourceMap": false,
     "lib": [
+      "dom",
       "es5",
-      "dom"
+      "es6"
```

Comment:
So the updated `@types/node` acquired by `npm run nuke` has a reference to `Set` and even though node types are not technically used by esm-amd-loader, because they are hoisted, they are processed.  Without including `es6` in the tsconfig here, I don't know how to suppress the error encountered when running tsc.  Adding `../../node_modules/@types/node` to the ignore block doesn't seem to work.

Reply:
well, you shouldn't ever need both es5 and es6 at least.

Reply:
Ok. I'll flip it to be *just* es6 then for now.